packages/test-data/tests-unit/operations-css-vars/operations-css-vars.less

```
@@ -0,0 +1,23 @@
+// min()/max() and bare operations holding a CSS var() cannot be computed at
+// build time, so they are preserved as-is for the browser (issue #3777).
+.min-max-var {
+  a: min(100% - var(--w), 10px);
+  b: max(var(--x), 40px);
+  c: min(50vh - var(--y), var(--z) + 2px, 40px);
+}
+.operation-var {
+  d: 100% - var(--w);
+  e: var(--x) + 2px;
+}
+// A nested operation is left for the browser too, and keeps the grouping its tree
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)